content/credits.html

```diff
@@ -0,0 +1,57 @@
+---
+title: Credits
+description: Many thanks to these amazing people for contributing to the project by working on Scratch Addons, its features, its themes, its website, etc.
+aliases:
+  - /contributors
+---
+
+{{< scss "assets/css/contributors.scss" >}}
+
+<h1>Credits</h1>
+
+<div class="container">
+
+	<section id="contributors" class="mb-3">
+
+		<h2>Contributors</h2>
+	
+		<p>Many thanks to these amazing people for contributing to the project by working on Scratch Addons, its features, its themes, its website, etc. (<a href="https://allcontributors.org/docs/en/emoji-key">emoji key</a>)</p>
+	
+		<p>Contributions of any kind welcome! Have contributed in any way? Read how to put your name <a href="https://github.com/ScratchAddons/contributors/issues/{{< specifics/contributors-issue >}}">here</a>.</p>
+	
+		<p class="mb-3">This project follows the <a href="https://github.com/all-contributors/all-contributors">all-contributors</a> specification. View the contributors repository <a href="https://github.com/ScratchAddons/contributors/">here</a>.</p>
+	
+		{{< specifics/contributors >}}
+	
+	</section>
+
+	<section id="dependencies" class="mb-3">
+
+		<h2>Dependencies</h2>
+
+		<p>These dependencies are being used to make Scratch Addons possible.</p>
+
+		{{< specifics/dependencies >}}
+
+	</section>
+
+	<section id="special">
+
+		<h2>Special Thanks</h2>
+
+		<p>Special thanks to everyone who is listed below.</p>
+
+		<ul>
+			<li>The Scratch Foundation for creating Scratch.</li>
+			<li>All of the Scratch userscript, userstyle, and other tool creators.</li>
+			<li>Griffpatch for jumpstarting Scratch Addons with his extensions.</li>
+			<li>Everyone who sends us tons of feedback.</li>
+			<li>Everyone who writes a review.</li>
+			<li>Everyone who contributes but haven't been listed.</li>
```

Comment:
Hasn't

Reply:
> `everyone` is multiple.

It's used to refer to multiple people, but it's singular (or the sentence would start with "everyone who contribute").

Reply:
> everyone is multiple

Is it really? Would you say `Everyone on the server are friendly`?

Reply:
Sentence above is: 

> Everyone who writes a review

Which is equivalent to:

> Everyone that writes a review

`Writes` is used instead of `write`, so "everyone" is assigned he/she/it on that sentence.

Reply:
Sorry for the ambiguous sentences. I'm pretty sure that "everyone" is a singular term, even though it refers to all people. I will fix this. Sorry.

Reply:
The fix has been pushed, with another extra entry, and the "you." Please check.